README.md

```diff
@@ -83,6 +83,14 @@ on changes in every new commit: formatting type-checking, tests, linting.
 ln -s ../../pre-commit .git/hooks/pre-commit
 ```
 
+### Building/running Docker image
+
+```shell
+export AUTOHOST_IMAGE_TAG=recoil-autohost
```

Comment:
nitpick: variable is a bit redundant, I expect it's more useful to have in shell history full command and not have to issue 2 separate ones, and we use it in only 2 invocations.